integration_tests/integration/incomingMovements.spec.ts

```diff
@@ -0,0 +1,32 @@
+import IncomingMovementsPage from '../pages/incomingMovements'
```

Comment:
Can we keep this ChoosePrisoner.spec.ts ? 

So we're consistent with naming across controllers, spec and html